src/s3/util/fileViewerManager.ts

```diff
@@ -224,7 +208,14 @@ export class S3FileViewerManager {
 
     private async createTemp(): Promise<void> {
         this.tempLocation = await makeTemporaryToolkitFolder()
-        showOutputMessage(`folder created with location: ${this.tempLocation}`, this.outputChannel)
-        getLogger().debug(`folder created with location: ${this.tempLocation}`)
+        showOutputMessage(
+            localize(
+                'AWS.s3.message.tempCreation',
+                'Temp folder for FileViewer created with location: {0}',
+                this.tempLocation
+            ),
+            this.outputChannel
+        )
+        getLogger().debug(`Temp folder for FileViewer created with location: ${this.tempLocation}`)
```

Comment:
For the most part, if you're outputting a message to the output channel, that's probably a good indication that the log statement isn't at a debug level. I would log this at `info` level; if there's other information that you'd like to surface here for debug purposes, you could then log _that_ at a lower level.